modyn/supervisor/internal/triggers/drift/detector/evidently.py

```diff
@@ -110,6 +118,7 @@ def _get_evidently_metrics(
             EmbeddingsDriftMetric(EVIDENTLY_COLUMN_MAPPING_NAME, _evidently_metric_factory(config)),
         )
         for metric_ref, config in metrics_config.items()
+        if config.id not in ["EvidentlyHellingerDistanceDriftMetric"]
```

Comment:
can we make this a global in this file like `SPECIAL_METRICS` or something?

Reply:
sure